tools/rdm/TestMixins.py

```diff
@@ -90,6 +111,18 @@ def VerifyResult(self, response, fields):
           (self.EXPECTED_FIELD.capitalize(), self.PID,
            fields[self.EXPECTED_FIELD].encode('string-escape')))
 
+    if self.MIN_LENGTH and len(fields[self.EXPECTED_FIELD]) < self.MIN_LENGTH:
```

Comment:
This can then become 

if self.MIN_LENGTH and len(string_field) < self.MIN_LENGTH: